examples/replica/postgresql_replica.json

```diff
@@ -71,7 +71,7 @@
       "name": "IMAGESTREAMTAG",
       "displayName": "ImageStreamTag",
       "description": "The OpenShift ImageStreamTag to use for PostgreSQL.",
-      "value": "postgresql:9.5"
+      "value": "postgresql:9.6"
```

Comment:
there's no postgres 9.6 imagestream tag in openshift yet:
https://github.com/openshift/origin/blob/master/examples/image-streams/image-streams-centos7.json#L724-L795

so you need to deliver that before you start making templates default to looking for it.

Reply:
We'll update at once - in our repos. The template will be actually tested with freshly build and tagged image into custom namespace.